### PR TITLE
[feat] Feat/showpages

### PR DESCRIPTION
**Pull request by marcoviappiani**

### CODE REVIEW

**Comment by codydaig** [comment]
LGTM

File: modules/core/client/controllers/pages.controller.client.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    $stateProvider
+      .state('pages', {
+        parent: 'nav',
+        abstract: true
+      });
+
+    var currPage;
+    var currPageState;
+
+    for(var i =0; i<pages.length; i++) {
```
Comment (RRDuke) [inline modules/core/client/controllers/pages.controller.client.js]: spaces between assignment and value?

**Comment by RRDuke** [comment]
LGTM

**Comment by trevorhanus** [comment]
LGTM

**Pull request merged**